Common subdirectories: a/colors and b/colors
Common subdirectories: a/faces and b/faces
diff -u a/gtkasset.cpp b/gtkasset.cpp
--- a/gtkasset.cpp	2004-06-04 03:26:55.000000000 -0700
+++ b/gtkasset.cpp	2012-04-21 22:14:53.537082591 -0700
@@ -21,6 +21,7 @@
 
 #include <glib.h>
 #include <stdio.h>
+#include <cstring>
 
 #include "gtkmain.h"
 #include "gtkasset.h"
diff -u a/gtkbigpage.cpp b/gtkbigpage.cpp
--- a/gtkbigpage.cpp	2004-06-04 03:26:55.000000000 -0700
+++ b/gtkbigpage.cpp	2012-04-21 22:26:34.480395350 -0700
@@ -22,6 +22,7 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <unistd.h>
+#include <cstring>
 
 #include "gtkbigpage.h"
 #include "gtkmain.h"
diff -u a/gtkbigpage.h b/gtkbigpage.h
--- a/gtkbigpage.h	2004-06-04 03:26:55.000000000 -0700
+++ b/gtkbigpage.h	2012-04-21 22:18:20.940409836 -0700
@@ -33,6 +33,8 @@
 gpointer BubbleWatcher(gpointer local);
 gpointer DLWatcher(gpointer local);
 
+class GtkBubble;
+
 class BigPage : PalCallback{
 public:
 	BigPage(GtkWidget * Notebook, GtkWidget * LogNotebook);
diff -u a/gtkbubble.cpp b/gtkbubble.cpp
--- a/gtkbubble.cpp	2004-06-04 03:26:55.000000000 -0700
+++ b/gtkbubble.cpp	2012-04-21 22:23:50.803733488 -0700
@@ -19,6 +19,7 @@
 
 #include "gtkbubble.h"
 #include "gtkmain.h"
+#include <cstring>
 
 GtkBubble::GtkBubble(BigPage * owner){
 	this->owner = owner;
